css/style.css

```diff
@@ -0,0 +1,198 @@
+* {
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    background-color: #ffffff;
+    font-family: 'Amatic SC' !important;
+}
+
+.aboutcolor{
+	background-color: #c7d8c6;
+
+}
+.sabrina {
+    height: 300px;
+    width: 300px;
+    margin: auto;
+    border: 20px solid white;
+    border-radius: 10px;
+}
+
+.intro {
+    font-family: 'Bungee Shade', cursive;
+    background-color: #fa7c92;
+    text-align: center;
+    padding-bottom: 20px;
+    padding-top: 20px;
+
+}
+
+.para {
+    font-family: 'Amatic SC', cursive;
+    background-color: #fa7c92;
+    padding-bottom: 20px;
+    padding-top:20px;
+    text-align: center;
+    display: block;
+    border-radius: 30px;
+    width: 300px;
+    border: 25px solid white;
+    padding: 25px;
+    margin: auto 0;
+    position: absolute;
+
+}
+
+.closing {
+    font-family: 'Amatic SC', cursive;
+    background-color: #c7d8c6;
+    text-align: center;
+    background-color: #fa7c92;
+    padding-bottom: 40px;
+    padding-top:30px;
+    text-align: center;
+    display: block;
+    border-radius: 30px;
+    width: 300px;
+    border: 25px solid white;
+   float: right;
+   clear: both;
+  
+}
+
+h1 {
+    font-family: 'Bungee Shade' !important;
+    background-color: #fa7c92;
+    padding: 30px;
+   
+}
+
+nav {
+    
+    overflow: hidden;
+    color: black;
+    padding: 20px;
+    padding-left: 100%;
+    float:right;
+    clear: both;
+    content:"";
+    margin: 0 auto;
+    background-color: #FFF7c0;
+    font-family:'Amatic SC'!important;;
+
+}
+
+nav a {
+    nav a:hover {
```

Comment:
You can't nest style rules within one another like this. You can in Sass, not in plain CSS.